snuba/datasets/transactions.py

```diff
@@ -75,10 +75,7 @@ def __init__(self):
             ('start_ms', UInt(16)),
             ('finish_ts', DateTime()),
             ('finish_ms', UInt(16)),
-            ('duration', Materialized(
-                UInt(32),
-                '((finish_ts - start_ts) * 1000) + (finish_ms - start_ms)',
-            )),
+            ('duration', UInt(32)),
```

Comment:
Kind of OT but how do we deal with migrations in Snuba, and is there anything we will need to provide for open source?

Reply:
We could add it to migrations.py though
- this is a relatively new dataset and only used for a product we did not launch yet, so I would not be too concerned. 
- running the sentry snuba tests automatically drops and recreate the table so just by running those test the schema fixes itself.

Reply:
I actually added the migration rule in the last commit.